addons/info/src/components/PropVal.js

```diff
@@ -51,13 +52,19 @@ function previewArray(val, maxPropArrayLength) {
   return <span style={valueStyles.array}>[{createFragment(items)}]</span>;
 }
 
-function previewObject(val, maxPropObjectKeys) {
+previewArray.propTypes = {
```

Comment:
`previewArray` is never rendered as a component, it's only called as a function. So looks like those propTypes aren't really checked at any point. You should either remove them, or capitalize the name (`PreviewArray`) and replace `previewArray(props)` below with `<PreviewArray {...props} />`.

If you decide to do the latter, please note that `val` should rather be `PropTypes.array`, and don't disable all of the eslint rules — only the particular ones (`// eslint-disable react/forbid-prop-types`)